sdc/tests/test_sdc_numpy.py

```diff
@@ -80,7 +62,7 @@ def ref_impl(a):
             return a.astype('float64')
 
         def sdc_impl(a):
-            return astype(a, 'float64')
+            return numpy_like.astype(a, 'float64')
```

Comment:
Do we need this test? Isn't it covered by `test_astype_to_num`?

Reply:
This tests the parameter that is passed from the jitted region